core/drivers/crypto/hisilicon/hpre_ecc.c

```diff
@@ -479,14 +479,63 @@ static enum hisi_drv_status ecc_dh_out_to_crypto_bin(struct hpre_ecc_msg *msg)
 	return ret;
 }
 
-static enum hisi_drv_status hpre_ecc_out_to_crypto_bin(struct hpre_ecc_msg *msg)
+static enum hisi_drv_status ecc_sign_out_to_crypto_bin(struct hpre_ecc_msg *msg)
+{
+	struct hpre_ecc_sign *ecc_sign = &msg->param_size.ecc_sign;
+	uint8_t *r = msg->out;
+	uint8_t *s = r + msg->key_bytes;
+	enum hisi_drv_status ret = HISI_QM_DRVCRYPT_NO_ERR;
+
+	ret = hpre_bin_to_crypto_bin(r, r, msg->key_bytes,
+				     ecc_sign->r_bytes);
+	if (ret) {
+		EMSG("Fail to transfer ecc sign r from hpre_bin to crypto_bin");
+		return ret;
+	}
+
+	ret = hpre_bin_to_crypto_bin(s, s, msg->key_bytes,
+				     ecc_sign->s_bytes);
+	if (ret)
+		EMSG("Fail to transfer ecc sign s from hpre_bin to crypto_bin");
+
+	return ret;
+}
+
+static enum hisi_drv_status hpre_ecc_verify_get_result(struct hpre_ecc_msg *msg,
+						       struct hpre_sqe *sqe)
+{
+	uint32_t result = (uint32_t)sqe->out;
```

Comment:
How about something like:
``` C
if (sqe->out & BIT64(1)) {
    msg->result = 1;
    return HISI_QM_DRVCRYPT_NO_ERR;
} else {
    msg->result = 0;
    return HISI_QM_DRVCRYPT_VERIFY_ERR;
}
```
The code could benefit from descriptive defines instead of the magic numbers `0` and `1`.